lib/ukrust/ukrust_sys/allocator.rs

```diff
@@ -50,35 +50,38 @@ unsafe impl GlobalAlloc for UkAlloc {
 #[global_allocator]
 static ALLOCATOR: UkAlloc = UkAlloc;
 
-#[no_mangle]
-pub fn __rust_alloc(size: usize, _align: usize) -> *mut u8 {
-    unsafe { bindings::__ukrust_sys_malloc(size as u64) as *mut u8 }
-}
+// #[no_mangle]
+// pub extern "Rust" fn __rust_alloc(size: usize, _align: usize) -> *mut u8 {
```

Comment:
Is this intended in this PR? I think without `__rust_alloc` we won't be able to allocate anything on the heap.

Reply:
In fact this is another issue I need to solve later.
This is a `multiple definition` problem. If enable these functions, rustc will report these errors. 
And the target of this PR aims to run `helloworld-rust` at least becasue `ukrust` cannot be used yet. Maybe the upgrade of rust cause this problem.
In https://github.com/unikraft/unikraft/issues/1147 is more detailed.